node/bft/src/bft.rs

```diff
@@ -602,7 +602,7 @@ impl<N: Network> BFT<N> {
                     // Retrieve the transmissions.
                     for transmission_id in certificate.transmission_ids() {
                         // If the transmission already exists in the map, skip it.
-                        if transmissions.contains_key(transmission_id.0) {
+                        if transmissions.contains_key(transmission_id) {
```

Comment:
The VM requires {transaction, solution}_ids to be unique, and maintains the same order as this for-loop, so its more efficient to skip transmissions by id here already.

Maybe noting that as a comment would also be helpful. O:)

Reply:
I'm not sure `transmission_id.0` even compiles. In addition, the subsequent call to `tx_consensus_subdag.send` requires the full transmission ID, including the checksum.

The checksum here is already provided, so this does not incur any additional overhead.